query/common/src/main/java/org/codice/ddf/admin/common/fields/base/BaseHiddenField.java

```diff
@@ -11,23 +11,27 @@
  * is distributed along with this program and can be found at
  * <http://www.gnu.org/licenses/lgpl.html>.
  **/
-package org.codice.ddf.admin.common.fields.base.scalar;
+package org.codice.ddf.admin.common.fields.base;
 
 import static org.codice.ddf.admin.api.fields.ScalarField.ScalarType.HIDDEN_STRING;
 import static org.codice.ddf.admin.common.report.message.DefaultMessages.emptyFieldError;
 
 import java.util.List;
 import java.util.Set;
 
+import org.codice.ddf.admin.api.fields.HiddenField;
 import org.codice.ddf.admin.api.report.ErrorMessage;
+import org.codice.ddf.admin.common.fields.base.scalar.BaseScalarField;
 import org.codice.ddf.admin.common.report.message.DefaultMessages;
 
 import com.google.common.collect.ImmutableSet;
 
-public class BaseHiddenField extends BaseScalarField<String> {
+public class BaseHiddenField extends BaseScalarField<String> implements HiddenField {
```

Comment:
Don't know if we caught this before but this should probably be abstract since it is meant as a base to other classes.